spec/vcr_setup.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+ENV["RAILS_ENV"] ||= 'test'
+
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
+  config.hook_into :webmock
+  config.default_cassette_options = { record: :once }
+end
```

Comment:
I think that this should go to `spec/support` and then it's loaded automatically. There are also a couple of very handy options to add:

* `ignore_localhost = true` for all system specs. https://relishapp.com/vcr/vcr/v/6-1-0/docs/configuration/ignore-request
* `configure_rspec_metadata!` means that you don't have to provide a name. https://relishapp.com/vcr/vcr/v/6-1-0/docs/test-frameworks/usage-with-rspec-metadata

Reply:
Awesome. Many thanks for the early review.

The `configure_rspec_metadata!` is working now - done on 62daf5643fbdf103b76eef261b6cf4f32af51954.

All we need to do is add the `:vcr` tag to the example, and it will create a cassette-file, based on the example name :+1: